demo/client/backend/mock/README.md

```diff
@@ -0,0 +1,35 @@
+# Start with go chaincode
+
+    $ go run .
+
+# Start with fpc chaincode
+
+Note that you need to start the docker container from the fpc root
+folder
+
+    $ docker run -p 3000:3000 -it -v $PWD:/project/src/github.com/hyperledger-labs/fabric-private-chaincode hyperledger/fabric-private-chaincode-dev
+    $ make clean && make
+    $ cd ecc
+    $ make sym-links
+
+We need some more go deps here
+
+    $ go get github.com/golang/protobuf/proto && go get golang.org/x/sync/semaphore
+    $ go get github.com/gin-contrib/cors && go get github.com/gin-gonic/gin
+
+Next build your FPC chaincode.
+    
+    $ cd $FPC_PATH/examples/YOUR_CHAINCODE
+    $ make
+
+We are almost there ...
```

Comment:
given that before this steps I compiled the chaincode, it was not immediately clear these had to be executed in the mock folder.

Reply:
ok, I add a `$ cd $FPC_PATH/demo/client/backend/mock` so it's clear where to continue after building the chaincode